src/druglab/db/table/molecule.py

```
@@ -464,75 +390,54 @@ def update_from_conformers(
-                    val = col_vals.min()
-                elif agg_fn == "max":
-                    val = col_vals.max()
-                elif agg_fn == "mean":
-                    val = col_vals.mean()
-                elif agg_fn == "sum":
-                    val = col_vals.sum()
-                elif callable(agg_fn):
-                    val = agg_fn(col_vals)
-                else:
-                    val = col_vals.iloc[0]
+                if agg_fn == "first":    val = col_vals.iloc[0]
```

Ruff (0.15.10)

[error] Multiple statements on one line (colon)

(E701)